.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: ruby
 cache: bundler
-sudo: false
+sudo: required
 
+before_install:
```

Comment:
Interesting, I hadn't seen that. Agree that if it slows down tests we should exclude it.